qemu/tests/usb_basic_check.py

```diff
@@ -20,6 +21,22 @@ def check_usb_device_monitor(test, params, env):
                              "returns: \n%s" % (params.get("product"), o))
 
 
+def check_usb_device_guest(session, item, cmd):
+    def __check(session, item, cmd):
+        output = session.cmd_output(cmd)
```

Comment:
Here seems also has the problem the the timeout in wait_for is shorter then the timeout inside.